docs/diagrams/DeleteStudentSequenceDiagram.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
```

Comment:
Hi! I would suggest adding this to your folder. Maybe deleting the old DeleteSequenceDiagram.png might be good also, if its still in the file.

Reply:
i think i actually screwed up with my master branch at some point, so will just redo/copy paste over the relevant changes when I manage to pull from our shared master